sirepo/job_driver/__init__.py

```diff
@@ -308,16 +327,30 @@ def _agent_receive_alive(self, msg):
         self._websocket_ready_timeout_cancel()
         if self._websocket:
             if self._websocket != msg.handler:
-                raise AssertionError(pkdformat("incoming msg.content={}", msg.content))
+                self._websocket = None
```

Comment:
This is the second place you've added this. Maybe we add a with that cleans up websocket?

Reply:
A "with" is not possible. This is a callback from tornado. I refactored in 6e0f1ee, lmk.